services/generator/service.go

```diff
@@ -54,6 +54,7 @@ func (s *service) GenerateTemplateData(params api.Params, currentReplicas int, g
 		AllHosts:            append(params.Hosts, params.InternalHosts...),
 		AllHostsJoined:      strings.Join(append(params.Hosts, params.InternalHosts...), ","),
 		IngressPath:         params.Basepath,
+		PathType:            params.Basepath,
```

Comment:
Better set `PathType: "Prefix"` so can skip doing that on line 468

Reply:
Done